packages/os/bootstrap-commands-toml

```diff
@@ -0,0 +1,12 @@
+[required-extensions]
+bootstrap-commands= "v1"
+std = { version = "v1", helpers = ["if_not_null", "join_nested_array"]}
++++
+{{#if_not_null settings.bootstrap-commands}}
+{{#each settings.bootstrap-commands}}
+[bootstrap-commands."{{@key}}"]
+commands = [{{join_nested_array this.commands}}]
```

Comment:
Why do we need the helper? Can't we just pass `this.commands`?

Reply:
I attempted. It didn't work.

Reply:
I think this has the capacity to output invalid `toml` -- if the inputs have any characters in `[]"`, then there may be problems.

I believe you can use `'` for raw strings in TOML, and triple-single-quotes for raw strings which allow `'` as well.

Might be a good idea to write a test case in the `bootstrap-commands` binary which checks against input like this.

Reply:
As an alternative to `join_nested_array`, I wonder if you could kill two birds with one stone and implement it in the template like this:

```
{{#each settings.bootstrap-commands}}
[bootstrap-commands."{{@key}}"]
commands = [
{{#each this.commands}}
    [{{#each this}}'''{{ this }}''',{{/each}}],
{{/each}}
]
{{/each}}
```

It seems to [work on the handlebars playground](https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22commands%20%3D%20%5B%5Cn%7B%7B%23each%20commands%7D%7D%5Cn%20%20%5B%7B%7B%23each%20this%7D%7D'''%7B%7B%20this%20%7D%7D'''%2C%7B%7B%2Feach%7D%7D%5D%2C%5Cn%7B%7B%2Feach%7D%7D%5Cn%5D%5Cn%22%2C%22partials%22%3A%5B%5D%2C%22input%22%3A%22%7B%20commands%3A%20%5B%5B%5C%22apiclient%5C%22%2C%20%5C%22get%5C%22%2C%20%5C%22settings.motd%5C%22%5D%2C%20%5B%5C%22apiclient%5C%22%2C%20%5C%22set%5C%22%2C%20%5C%22settings.motd%3Dhi%5C%22%5D%5D%20%7D%5Cn%22%2C%22output%22%3A%22commands%20%3D%20%5B%5Cn%20%20%5B'''apiclient'''%2C'''get'''%2C'''settings.motd'''%2C%5D%2C%5Cn%20%20%5B'''apiclient'''%2C'''set'''%2C'''settings.motd%26%23x3D%3Bhi'''%2C%5D%2C%5Cn%5D%5Cn%22%2C%22preparationScript%22%3A%22Handlebars.registerHelper('loud'%2C%20function%20(aString)%20%7B%5Cn%20%20%20%20return%20aString.toUpperCase()%5Cn%7D)%5Cn%22%2C%22handlebarsVersion%22%3A%224.7.8%22%7D), other than the unusual URL encoding. Also seems to parse properly per [this site](https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22commands%20%3D%20%5B%5Cn%7B%7B%23each%20commands%7D%7D%5Cn%20%20%5B%7B%7B%23each%20this%7D%7D'''%7B%7B%20this%20%7D%7D'''%2C%7B%7B%2Feach%7D%7D%5D%2C%5Cn%7B%7B%2Feach%7D%7D%5Cn%5D%5Cn%22%2C%22partials%22%3A%5B%5D%2C%22input%22%3A%22%7B%20commands%3A%20%5B%5B%5C%22apiclient%5C%22%2C%20%5C%22get%5C%22%2C%20%5C%22settings.motd%5C%22%5D%2C%20%5B%5C%22apiclient%5C%22%2C%20%5C%22set%5C%22%2C%20%5C%22settings.motd%3Dhi%5C%22%5D%5D%20%7D%5Cn%22%2C%22output%22%3A%22commands%20%3D%20%5B%5Cn%20%20%5B'''apiclient'''%2C'''get'''%2C'''settings.motd'''%2C%5D%2C%5Cn%20%20%5B'''apiclient'''%2C'''set'''%2C'''settings.motd%26%23x3D%3Bhi'''%2C%5D%2C%5Cn%5D%5Cn%22%2C%22preparationScript%22%3A%22Handlebars.registerHelper('loud'%2C%20function%20(aString)%20%7B%5Cn%20%20%20%20return%20aString.toUpperCase()%5Cn%7D)%5Cn%22%2C%22handlebarsVersion%22%3A%224.7.8%22%7D).

Reply:
I solved it with triple curly braces for now but I would like to spend some more time on this to think about a better way to do this to tackle corner cases that we discussed.